### ISSUE TITLE
Error on get access token 

### ISSUE THREAD

**Issue by master-7** [yandex-money/yandex-money-sdk-nodejs#3]
First step - obtain token URL is work, but second step - get access token is failed - error: "invalid_request".
clientId, code, redirectURI - true.

**Comment by master-7** [comment]
```
tokenComplete = (err, data) => {
                if(err) {
                    done(null, err);
                    return;
                }
                if(!data.access_token) {
                    done(null, data);
                    return;
                }
                done(null, data.access_token);
            };

            yandexMoneySdk.Wallet.getAccessToken(
                yandexMoneyConfig.clientId,
                code,
                yandexMoneyConfig.redirectURI,
                null,
                tokenComplete
            );
```

**Comment by master-7** [comment]
Field grant_type: "authorization_code" lost in 
```
Wallet.getAccessToken = function (clientId, code, redirectURI, clientSecret,
    callback) {
  var full_url = base.SP_MONEY_URL + "/oauth/token";
  request.post({
    "url": full_url,
    "form": {
      "code": code,
      "client_id": clientId,
      "redirect_uri": redirectURI,
      "client_secret": clientSecret
    }
  }, base.processResponse(callback)
  );
 
};
```

**Comment by yalovek** [comment]
Thank you! We approved pull request with fix and it's already in master.

**Comment by abaranovskii** [comment]
@yalovek could you please publish the latest version into npm? The current version 0.1.6 doesn't contain this fix.

**Comment by amel-true** [comment]
@abaranovskii thanks for reply. We published version 0.1.7
